packages/web-components/@groupby/elements-autocomplete/test/unit/autocomplete.test.ts

```diff
@@ -92,6 +92,18 @@ describe('Autcomplete Component', () => {
       expect(requestInitialData).to.be.called;
     });
 
+    it('should not request initial data if the component is already initialized', () => {
+      autocomplete._initialized = true;
+
+      autocomplete.connectedCallback();
+
+      expect(windowAddEventListener).to.not.be.calledWith(
```

Comment:
Does this not fit on one line? That said, consistency with surrounding code says that this should stay. No action needed.

Reply:
I went with consistency on the surrounding lines.